docs/src/main/asciidoc/application-lifecycle-events-guide.adoc

```diff
@@ -94,10 +94,18 @@ TIP: The events are also called in _dev mode_ between each redeployment.
 
 NOTE: The methods can access injected beans. Check the {quickstarts-blob-url}/application-lifecycle-events/src/main/java/org/acme/events/AppLifecycleBean.java[AppLifecycleBean.java] class for details.
 
+=== What is the difference from `@Initialized(ApplicationScoped.class)` and `@Destroyed(ApplicationScoped.class)`
+
+In the JVM mode, there is no real difference, except that `StartupEvent` is always fired *after* `@Initialized(ApplicationScoped.class)` and `ShutdownEvent` is fired *before* `@Destroyed(ApplicationScoped.class)`. 
+For a native executable build, however, `@Initialized(ApplicationScoped.class)` is fired as *part of the native build process*, whereas `StartupEvent` is fired when the native image is executed.
```

Comment:
`native executable build, however` -> `native executable build however` (comma isn't really needed)

Reply:
Well, I'm not an expert, but when using `however` as an adverb  commas are probably ok: https://www.oxfordlearnersdictionaries.com/definition/english/however#however_sng_3 :-)

Reply:
Yeah I'm not neither, so your fancy link beats my suggestion :P :P :P

Reply:
Good. I was ready to add more fancy links!